src/coefficients/StdFunctionCoefficient.hpp

```diff
@@ -0,0 +1,167 @@
+// Copyright (c) 2019-2020, Lawrence Livermore National Security, LLC and
+// other Serac Project Developers. See the top-level LICENSE file for
+// details.
+//
+// SPDX-License-Identifier: (BSD-3-Clause)
+// # Author: Jonathan Wong @ LLNL.
+
+/**
+    @file StdFunctionCoefficient.hpp
+ */
+
+#ifndef STD_FUNCTION_COEFFICIENT_HPP
+#define STD_FUNCTION_COEFFICIENT_HPP
+#include <functional>
+#include "mfem.hpp"
+
+using namespace mfem;
+
+/**
+   \brief StdFunctionCoefficient is an easy way to make an mfem::Coefficient using a lambda
+
+   This is a place holder until the coefficient of the same name is merged into mfem.
+
+*/
+class StdFunctionCoefficient : public Coefficient
+{
+public:
+  /// Constructor that takes in an mfem Vector representing the coordinates and produces a double
+  StdFunctionCoefficient(std::function<double(mfem::Vector &)> func);
+  virtual ~StdFunctionCoefficient() {}
```

Comment:
Do you plan on someone inheriting this class? Otherwise the virtual is not necessary.

Reply:
I don't. Do you think I should prohibit inheriting from this class?

Reply:
I say remove it for now and we can address inheritance when it comes up.  Or at least when we (and i really mean not me) sit down and figure a use case...